docs/src/.vitepress/theme/style.css

```diff
@@ -171,4 +171,38 @@ mjx-container > svg {
   --vp-c-brand-3: #91dd33;
   --vp-c-sponsor: #91dd33;
   --vitest-c-sponsor-hover: #e51370;
-}
\ No newline at end of file
+}
+
+
+.VPDoc.has-aside .content-container {
+  max-width: 100% !important;
+}
+.aside {
+  max-width: 200px !important;
+  padding-left: 0 !important;
+}
```

Comment:
```suggestion
```
this removes the hard coded size for the right panel, however, it goes back to the default size, which is quite big for my taste. On the other hand we might recover the proper alignment.